docs/intentReaderHandlers/RegisterEventHandler.md

```diff
@@ -0,0 +1,357 @@
+# RegisterEventHandler 
+
+## Overview
```

Comment:
This handler is used to register for the event in FCA. should be changed to 
This handler is used to register an event in FCA.

Reply:
updated